ui/components/content-renderers/footnote-citation-in-table.js

```diff
@@ -6,7 +6,7 @@ import Link from '../link';
 export default function FootnoteCitationInTable({ content }) {
   const href = `#${content.footnote_node.identifier}-table`;
   return (
-    <sup className="footnote-citation-in-table">
+    <sup className="footnote-citation-in-table" id={`citation-${content.text}`}>
```

Comment:
The text of a footnote citation and the marker of the footnote it refers to might be slightly different. Can we use `content.footnote_node.identifier` in both, similar to line 7?

Reply:
Definitely agree on this. Updated.